README.md

```diff
@@ -1,2 +1,45 @@
-# new-relic
-Best practice New Relic configurations.
+# New Relic considerations
+
+Considerations for using New Relic in the federal government.
+
+Below is a description of the settings in your New Relic `.ini` file that concern privacy and security. Example configuration files can be found in this repository:
+
+* [`python-low-security.ini`](python-low-security.ini)
```

Comment:
Since New Relic changes their generated config file relatively frequently with new releases of the agent, not sure that it's useful to provide one here that we need to keep up-to-date.